src/service/resource.js

```diff
@@ -626,6 +626,9 @@ export class Resource {
    * Returns a previously measured layout box adjusted to the viewport. This
    * mainly affects fixed-position elements that are adjusted to be always
    * relative to the document position in the viewport.
+   * The returned layoutBox is
+   * relative to the top of the document for non fixed element,
+   * relative to the viewport for fixed element.
```

Comment:
I'm not sure how to describe this one, but it's not strictly relative to the viewport. The best that I can think of is "relative to the top of the document _at the current scroll position_".

Eg, a fixed-position element is `top: 100px`. Then with 0 scroll, it'll be `{ top: 100 }`. At 20 scroll, it'll be `{ top: 120 }`.

Reply:
I see. It's also relative to the top of the document, but the the layoutbox is continuously changing with user scroll so that the fixed element position is unchanged. 
I like the "relative to the top of the document at the current scroll position" I'll switch to that.

Reply:
done